spec/requests/users_spec.rb

```diff
@@ -1,7 +1,39 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
+RSpec.describe 'Posts', type: :request do
```

Comment:
- [ ] This request should belong to users not posts, please change the name and the tests to ensure you are testing the index and show actions of users_controller as per the requirement.